### PR TITLE
Danish translation

### PR DESCRIPTION
**Pull request by Danielss89**

### CODE REVIEW

File: languages/da/Zend_Captcha.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+
+/**
+ * EN-Revision: 30.Jul.2011
```
Comment (Danielss89) [inline languages/da/Zend_Captcha.php]: Should this be the date i translated, or taken from the english translation?

File: languages/da/Zend_Validate.php [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+
+/**
+ * EN-Revision: 16.Jun.2015
```
Comment (Danielss89) [inline languages/da/Zend_Validate.php]: Should this be the date i translated, or taken from the english translation?

File: languages/da/Zend_Captcha.php (hunk 1 above)
Comment (samsonasik) [inline languages/da/Zend_Captcha.php]: date we modified iirc, see 'id' translation: https://github.com/zendframework/zend-i18n-resources/blob/master/languages/id/Zend_Captcha.php#L11

**Comment by Danielss89** [comment]
@Maks3w @weierophinney ETA of merging of this?

**Pull request merged**